apps/mobile/src/libs/styles/global-styles/flex.ts

```diff
@@ -0,0 +1,34 @@
+const FLEX = {
```

Comment:
```suggestion
import type { FlexStyle } from 'react-native';

type GlobalFlexStyles = {
  [key: string]: FlexStyle;
};

const FLEX = {
```

Reply:
Maybe type GlobalFlexStyles should be moved to the types/styles folder?